moped-editor/src/views/projects/projectView/ProjectActivityEntry.js

```diff
@@ -10,22 +10,27 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const ProjectActivityEntry = ({
-  changeIcon,
-  changeDescription,
-  changeValue,
-}) => {
+/**
+ * One row/entry in the project activity log
+ * @param {changeIcon} Material UI icon component
+ * @param {changeText} - Array of objects of the shape:
+ *  {text: String to display, style: String name of style or null}
+ */
```

Comment:
would love feedback on how to make this better

Reply:
i really like this pattern!

Reply:
this make sense to me! so every item in the array can have its own styling

Reply:
Same for me! I like this way of representing the different pieces of the activity log text.